packagehandlers/conanpackagehandler.go

```diff
@@ -0,0 +1,94 @@
+package packagehandlers
+
+import (
+	"errors"
+	"fmt"
+	"github.com/jfrog/frogbot/v2/utils"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	conanTechName = "conan"
+	conanFileTxt  = "conanfile.txt"
+	conanFilePy   = "conanfile.py"
+)
+
+type ConanPackageHandler struct {
+	CommonPackageHandler
+}
+
+func (conan *ConanPackageHandler) UpdateDependency(vulnDetails *utils.VulnerabilityDetails) error {
+	if vulnDetails.IsDirectDependency {
+		return conan.updateDirectDependency(vulnDetails)
+	} else {
+		return &utils.ErrUnsupportedFix{
+			PackageName:  vulnDetails.ImpactedDependencyName,
+			FixedVersion: vulnDetails.SuggestedFixedVersion,
+			ErrorType:    utils.IndirectDependencyFixNotSupported,
+		}
+	}
+}
+
+func (conan *ConanPackageHandler) updateDirectDependency(vulnDetails *utils.VulnerabilityDetails) (err error) {
```

Comment:
In general, as we talked - lets figure out if the 'installation' after the manual dependency fix is necessary.
If not - lets take it off to prevent unnecessary bugs.
If yes- We need to take care of Artifactory resolution and file cleanups before pushing

Reply:
as decided the installation command was removed